### PR TITLE
Update docker-compose.yml

### PR DESCRIPTION
**Pull request by alxjand**
The identification image version have been updated.
some environment variables that are no longer required in the identification and categorization modules have been removed

### CODE REVIEW

File: docker-compose/docker-compose.yml [hunk 1]
```diff
@@ -206,10 +204,6 @@ services:
      expose:
        - 46070
      privileged: true
-     environment:
```
Comment (cjdcordeiro) [inline docker-compose/docker-compose.yml]: you've removed these lines...are you sure they are not needed? did you double check with @syc17 ?

Comment (syc17) [reply docker-compose/docker-compose.yml]: Sorry, I don't have the context to the change.  My components do not call identification at the moment.  I am still waiting for instructions from UPC regarding the CAU call to validate IDKey.

Comment (alxjand) [reply docker-compose/docker-compose.yml]: When I read your message I originally thought I had made a mistake, but now that I see the [original file](https://github.com/mF2C/mF2C/blob/master/docker-compose/docker-compose.yml) I think I did not... lines 210-212 are in the resource-categorization section, so I guess I dont need to double check this with @syc17 (do i need to?) but with @ssgUPC and I did it.

Comment (alxjand) [reply docker-compose/docker-compose.yml]: Don't worry :) and yes, I already confirmed it with @ssgUPC

**Pull request merged**